core/src/scss/utilities/mixins/modular-scale/_modular-spacing-new.scss

```diff
@@ -0,0 +1,77 @@
+@charset "UTF-8";
+
+///
+/// Modular Spacing New
+/// Mixins for modular scaling, a unitless system for returning font sizes or spacing (padding or margin) values.
+/// Our modular spacing system does not use the Bourbon modular-scale mixin. Instead we use a manual scale defined in variables/core/_modular-spacing.scss for finetuning.
+/// A mixin for adding spacing (margin or padding) between elements or between an
+/// element and the edge of its container. The amount of spacing is returned
+/// using a unitless scaling system with a curated scale.
+/// Please see core/scss/utilities/variables/core/_modular-spacing.scss for
+/// variable maps.
+///
+/// @name modular-spacing-new
+///
+/// @example scss
+///   @include modular-spacing-new('padding', 2 null);
+///   @include modular-spacing-new('margin', 0 -4 1 2);
+///
+/// @param {string} $type  Spacing type, either 'padding' or 'margin'.
+/// @param {list} $steps - List (1 to 4 values): Either integer (from -4 to 10 including 0) or null. Follows the shorthand notation of padding/margin. Use null to skip a side. A value of 0 returns the modular spacing base value for that breakpoint.
+/// @param {map} $grid - Breakpoint device sass map. Defaults to $su-grid-device.
+///
+/// @group mixin
+@mixin modular-spacing-new($type: 'padding', $steps: 0, $grid: $su-grid-device) {
```

Comment:
I'd much rather not have a `-new` function but instead, change the original. We do want to change the spacing of everything (if even just a little). We can make this a minor release and have release notes about how to upgrade. If the change in the scale is too drastic and we cannot change this function to use the new values then we must deprecate it and re-name this function something else as `-new` gets old quick. Perhaps just `modular-space`

Reply:
I can see how the `-new` can get old real quick 😂   I'm just a little afraid of replacing it right away, since it's used everywhere including our masthead, global footer etc.

Reply:
Btw, I believe @kerri-augenstein said these values still need to be finetuned. I supposed it would be interesting to just replace the mixin in the branch and see what break and what work well, and tweak accordingly 🤔

Reply:
Double-checking with @kerri-augenstein here then. We DO want to change the modular spacing values everywhere we used them to the new ones? I guess my question is, how much different is say, a 4, from the previous function to the new one? If the adjustment is too big or a 4 yesterday should really be 5 tomorrow we need to talk about the DX of this change. We can move SOME cheese in this version, but we might have to run with the new function and deprecate the existing.

Reply:
Oh, some of the changes are pretty drastic (more so for the larger steps). But yeah, I anticipate some back and forth with @kerri-augenstein here.

Reply:
@sherakama I pushed up a branch `modular-spacing-map-replace`, which straight up replace the old mixin with this new one. I'm testing it on WP Redwood....strangely, working surprisingly well. Not really breaking things except in a few expected places (that's specific to that theme). Need more testing of course. Feel free to pull that branch and test it out on D8.